src/util/screening-selector.js

```diff
@@ -79,82 +78,83 @@ export function getInstrumentListFromCarePlan(
       qId = a.detail.instantiatesCanonical[0].split("/")[1];
     }
     if (!qId) return true;
+    if (instrumentList.indexOf(qId) === -1) instrumentList.push(qId);
 
-    // get matched completed questionnaire response(s) by questionnaire id
-    const qResults = responses.filter((q) => {
-      const questionnaireIdentifier = q.questionnaire.toUpperCase();
-      return (
-        q.status === "completed" &&
-        questionnaireIdentifier.indexOf(qId.toUpperCase()) !== -1
-      );
-    });
+    // // get matched completed questionnaire response(s) by questionnaire id
+    // const qResults = responses.filter((q) => {
+    //   const questionnaireIdentifier = q.questionnaire.toUpperCase();
+    //   return (
+    //     q.status === "completed" &&
+    //     questionnaireIdentifier.indexOf(qId.toUpperCase()) !== -1
+    //   );
+    // });
 
-    // check instruments against scheduled scheduledTiming
-    const scheduledTiming = detailElement.scheduledTiming;
-    const repeat = scheduledTiming ? scheduledTiming.repeat : null;
-    // check repeat schedule
-    const period =
-      repeat.period && !isNaN(repeat.period) ? parseInt(repeat.period) : 0;
-    // h | d | wk | mo, https://fhir-ru.github.io/datatypes.html#Timing
+    // // check instruments against scheduled scheduledTiming
```

Comment:
@mcjustin  I moved the scheduled timing logic to CQL (see my commit [here](https://github.com/uwcirg/asbi-screening-app/pull/76/commits/08ecef6584c6ae9772cef9a1afdbe8343e6137e3)) so that all the logic for determining which questionnaire(s) to administer now sits in one place, CQL logic library.  Let me know if you see any issue with my change.